stock_inventory_lockdown/models/stock_move.py

```diff
@@ -29,8 +29,8 @@ def _check_locked_location(self):
             reserved_locs = move._get_reserved_locations()
             dest_locs = move._get_dest_locations()
             if (locked_location_ids and
-                    move.product_id.property_stock_inventory not in [
-                        move.location_dest_id, move.location_id] and
+                    not any([move.location_dest_id.usage == 'inventory',
```

Comment:
I think this should not be in this PR

Reply:
Is in https://github.com/OCA/stock-logistics-warehouse/pull/716, but I put here also because I wanted a green travis-ci!

Reply:
Ok, in PR description, put 'Includes #716' instead of depends

Reply:
Well, my idea is wait until the other Pr is merged, and then rebase this one.

Reply:
Yeah I've understood that. My remark just to help reviewer :smiley: